Assets/Scripts/Game/InputManager.cs

```diff
@@ -1199,6 +1199,28 @@ public KeyCode ParseKeyCodeString(String s)
             }
         }
 
+        /// <summary>
+        /// Calculates UI scroll movement value based on analog axes input.
+        /// </summary>
+        public float GetUIScrollMovement()
+        {
+            float horizontal = Input.GetAxis(cameraAxisBindingCache[0]);
+            float vertical = Input.GetAxis(cameraAxisBindingCache[1]);
+
+            if (GetAxisActionInversion(AxisActions.CameraHorizontal))
+                horizontal *= -1;
+
+            if (GetAxisActionInversion(AxisActions.CameraVertical))
+                vertical *= -1;
+
+            float calculated_deadzone = Mathf.Sqrt(horizontal * horizontal + vertical * vertical);
```

Comment:
I'd rename this to camelCase format (`calculatedDeadzone`)

Reply:
Done